addons/AsepriteWizard/animation_player/animation_creator.gd

```diff
@@ -107,16 +107,36 @@ func _configure_animations(target_node: Node, player: AnimationPlayer, content:
 
 func _add_animation_frames(target_node: Node, player: AnimationPlayer, anim_name: String, frames: Array, context: Dictionary, direction = 'forward'):
 	var animation_name = anim_name
+	var library_name = _DEFAULT_ANIMATION_LIBRARY
 	var is_loopable = _config.is_default_animation_loop_enabled()
 
+	var anim_tokens := anim_name.split("/")
+
+	if anim_tokens.size() > 2:
+		push_error("Invalid animation name: %s" % animation_name)
```

Comment:
Given this is gonna break existing animations which don´t fit the format, I think we can give the option for people to disable it.

You can follow the `is_loopable` example on line 111. It should be a project setting disable by default.

Reply:
Any animation name with more than 1 slash is invalid and will cause Godot to error regardless (at least in Godot 4), so I believe this is necessary.

Reply:
Oh! I see. I wasn't aware of it. Good one. In that case this is also a necessary fix. Thank you.